tools/validate_spec.py

```diff
@@ -0,0 +1,332 @@
+#!/usr/bin/env -S python3 -O
```

Comment:
The current code looks good to me. What about the two unchecked tasks of #7?

Reply:
Added a test as described in https://github.com/Componolit/RecordFlux-specifications/issues/7 in 9779775. 
Since no manual interaction should be required,  have found no good generic way of specifying the message that should be validated out of  a package where more than one message is present (e.g. IPv4 with 'Frame' and 'Option').

Reply:
I am not sure I understand your issue. As far as I can see, the new test function iterates over all messages of package. What do you think is missing?

Reply:
Yes, correct. I did that because of the issue described above. I was not sure if it is useful to validate the `Ipv4::Option` Spec with standalone `Option` messages since they are never sent outside of an ipv4 packet.

Reply:
In case this is the expected behaviour, the unchecked tasks should be fulfilled.

Reply:
Now I see what you mean. I don't think that is a problem. I think it makes sense to validate each message separately.